.github/workflows/generate-preview-link.yml

```diff
@@ -189,3 +192,9 @@ jobs:
                   number: ${{steps.pr_information.outputs.issue_number}}
                   message: ${{steps.generate_preview_url.outputs.comment || steps.generate_failure_comment.outputs.comment }}
                   recreate: true
+
+            - name: Upload PR information to artifact
+              uses: actions/upload-artifact@v2
+              with:
+                  name: 'pr-${{github.run_id}}'
+                  path: .pr
```

Comment:
Uploads the artifacts so that it can be used in lighthouse and smoketest workflows